docs/_extra/api-reference/schemas/annotation.yaml

```diff
@@ -0,0 +1,87 @@
+Annotation:
```

Comment:
There is some duplication between the schemas in annotation.yaml and annotation-create.yaml. Is it possible to share the definitions for common fields?